src/fdTransfer_client.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2021 Yonatan Goldschmidt
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _FDTRANSFER_H
+#define _FDTRANSFER_H
+
+#ifdef __linux__
+
+#include "fdTransfer_shared_linux.h"
+
+class FdTransferClient {
+  private:
+    static int _peer;
+    static unsigned int _request_id;
+
+    static unsigned int nextRequestId() { return _request_id++; }
```

Comment:
Presently the reason to keep it is to avoid mixing perf requests with kallsyms requests. I'll modify it to a "type" field so we can check that instead.